docs/source/environments/braintumor.rst

```diff
@@ -0,0 +1,163 @@
+.. _braintumor:
+
+
+Brain Tumor PDE
+================
+
+This documentation describes the Brain Tumor DPR (Diffusion-Proliferation-Radiation) Environment, which models the growth dynamics of glioblastomas–a fast-growing type of brain cancer–and their response to external beam radiation therapy (XRT). The model is governed by a partial differential equation (PDE) that includes three key terms: diffusion of tumor cells, proliferation, and radiation-induced cell death.
+
+While tumor growth naturally occurs in three spatial dimensions and time, the model assumes spherical symmetry, allowing the simulation to be reduced to a one-dimensional radial domain over time. To further ease computation, the PDE is nondimensionalized, rescaling spatial, temporal, and concentration variables into a unitless form. As a result, the tumor’s evolving state can be represented as a one-dimensional spatial array indexed by time.
+
+The numerical methods and modeling framework are based on the methods proposed in [1]_ and [2]_.
+
+
+Background
+------------------------
+
+Glioblastoma is the most common malignant primary brain tumor, accounting for approximately 16% of all primary brain tumors [3]_. In contrast, the most common primary brain tumor overall–meningioma–makes up over 30% of primary brain tumor cases but tend to be benign and slow-growing [4]_. 
+
+Glioblastomas are fast-growing and highly aggressive, with a median survival time of only 15 months following diagnosis [3]_. Current standard-of-care involves a uniform, protocol-driven approach involving surgical resection (where feasible), external beam radiation therapy (XRT), and concurrent chemotherapy. Magnetic Resonance Imaging (MRI) is used throughout treatment to monitor tumor progression and response to therapy.
+
+Despite the known variability in tumor growth dynamics across patients, clinical radiation therapy regimens remain largely standardized. This “one-size-fits-all” approach limits the ability to tailor treatment to individual tumor dynamics. Computational models such as the DPR framework aim to simulate tumor evolution under different protocols, offering a pathway toward more personalized therapy strategies.
+
+
+
+Dimensional Model
+------------------------
+
+Let :math:`\mathcal{B} = [0, L]` denote the one-dimensional spatial domain of the brain, where :math:`x=0` corresponds to the tumor center and :math:`x=L` is the outer boundary of the simulation domain. This domain models radial tumor expansion under the assumption of spherical symmetry. 
+
+Let :math:`\mathcal{T} = [0, T]` denote the time domain of the simulation. Let :math:`\mathcal{T}_{\text{therapy}} \subset \mathcal{T}` represent the set of discrete time points at which radiation therapy is administered. The tumor concentration :math:`c(x, t)` evolves according to the following PDE:
```

Comment:
\mathcal{T}_{\text{therapy}} is not necesarrily discrete. For example, If I take the interval (0, 1), this is a continuous, fully dense interval on the real line. When one uses the word discrete, they mean individual numbers. All of the modelling in this section is done in continuous time. It is not until you apply the finite difference scheme that things become discrete.

Reply:
Should also be subset eq. No reason treatment isn't given for all time.